message/validation/genesis/consensus_validation.go

```diff
@@ -203,6 +207,7 @@ func (mv *messageValidator) validateSignerBehaviorConsensus(
 ) error {
 	signerState := state.GetSignerState(signer)
 
+	// MSGVALIDATIONREVIEW: validateJustifications is also called in the ending of this function.
```

Comment:
Being fixed here: https://github.com/ssvlabs/ssv/pull/1488

Reply:
The idea was that if there's no state, then we don't need all state-related checks because it's the first message for the new state and all checks would pass, but we still need to check justifications

Reply:
Oh ok. Got it. Will update

Reply:
Yes, I agree. I've added a comment in the fix PR